cmake/cpack_settings.cmake

```diff
@@ -22,4 +24,11 @@ set(CPACK_DEBIAN_OEHOSTVERIFY_PACKAGE_DEPENDS "")
 set(CPACK_DEBIAN_OEHOSTVERIFY_PACKAGE_RECOMMENDS "pkg-config")
 set(CPACK_DEBIAN_OEHOSTVERIFY_FILE_NAME DEB-DEFAULT)
 set(CPACK_COMPONENT_OEHOSTVERIFY_DESCRIPTION "Open Enclave Report Verification Host Library")
+
+# CPack variables for Nuget packages
+set(CPACK_NUGET_PACKAGE_NAME "open-enclave")
+set(CPACK_NUGET_PACKAGE_AUTHORS "Microsoft, Confidential Computing Consortium")
```

Comment:
🤔

Reply:
I think there may be a hidden benefit to reversing the authors naming order and I would recommend it, but it is a nit.

If authors are read as "Microsoft AND C3" it sounds like it is being branded by Microsoft and then C3 is adding to it Microsoft's efforts.

However, if it reads the reverse, it reads as if Microsoft is adding to C3's effort which I think might look better on paper as we are not looking to impose Microsoft's brand, per se.

Reply:
Should we just have "Confidential Computing Consortium" for this?

Reply:
I'd like to toss in a vote for calling it C3.

Further, if we were to do both, I would consider mentioning Confidential Computing Consortium before Microsoft.

Reply:
Andy and I talked about it, and we decided to rename this AUTHORS field to "Open Enclave SDK Contributors", because technically that is who will be responsible for the release packages.